src/main/java/Application.java

```diff
@@ -7,10 +17,20 @@ public static void main(String[] args) {
         OutputView outputView = new OutputView();
         InputView inputView = new InputView(new Scanner(System.in));
 
+        ConnectionProvider connectionProvider = new JanggiConnectionProvider();
+        GameDaoImpl gameRepository = new GameDaoImpl(connectionProvider);
+        PlayerDaoImpl playerRepository = new PlayerDaoImpl(connectionProvider);
+        PieceDaoImpl pieceRepository = new PieceDaoImpl(connectionProvider);
```

Comment:
변경된 클래스 명에 맞춰 객체 이름도 변경되면 좋을 것 같아요!

그리고 구현체를 통해 변수를 할당하면 인터페이스를 활용한 이점을 누릴 수 없을 것 같아요. 

```suggestion
        ConnectionProvider connectionProvider = new JanggiConnectionProvider();
        GameDao gameDao = new GameDaoImpl(connectionProvider);
        PlayerDao playerDao = new PlayerDaoImpl(connectionProvider);
        PieceDao pieceDao = new PieceDaoImpl(connectionProvider);
```

왜 구현체가 아닌 추상화된 인터페이스에 의존하는 것을 지향하는지 잘 고민해보면 좋을 것 같습니다!